tools/SwaggerCI/gitignoreconf

```diff
@@ -0,0 +1,5 @@
+bin
```

Comment:
What's this file for?

Reply:
This file is used to override the autorest generated .gitignore file. We want to commit the generated code, while the generated one will ignore it.